spec/models/manageiq/providers/base_manager/metrics_capture_spec.rb

```diff
@@ -121,4 +121,44 @@
       end
     end
   end
+
+  describe ".perf_capture_queue" do
+    before do
+      MiqRegion.seed
+      FactoryBot.create(:host_target_vmware, :ext_management_system => ems)
+
+      MiqQueue.delete_all
+
+      vm.perf_capture_realtime_now
+    end
+
+    let(:vm) { Vm.first }
```

Comment:
This looked really weird at first, I see now that the `host_target_vmware` factory creates 2 VMs.
Can you maybe do
```
let(:host) { FactoryBot.create(:host_target_vmware, ...)}
let(:vm) { host.vms.first}
```
So it is a little more obvious that the vm is coming from the host being created?